app/src/lib/design-system/components/RQEditorTitle/index.tsx

```diff
@@ -54,97 +56,104 @@ export const RQEditorTitle: React.FC<TitleProps> = ({
+                          onClick={() => setIsDescriptionEditable(true)}
+                          className="edit-description-btn"
+                          type="text"
+                        >
+                          Edit description
+                        </RQButton>
+                      ) : (
+                        <></>
+                      ),
+                    tooltip: false,
+                  }}
+                  onClick={() => setIsDescriptionEditable(true)}
+                >
+                  {description ? description : descriptionPlaceholder}
+                </Typography.Paragraph>
               </div>
-            </div>
-          ) : (
-            <div className="editor-title">
-              <Typography.Text
-                ellipsis={true}
-                onClick={() => {
-                  setIsNameEditable(true);
-                }}
-              >
-                {name ? name : namePlaceholder}
-              </Typography.Text>
-              <BiPencil onClick={() => setIsNameEditable(true)} />
-            </div>
-          )}
-        </Row>
-        <Row className="editor-title-description">
-          {isDescriptionEditable ? (
-            <TextArea
-              ref={textAreaRef}
-              autoSize={{ minRows: 1, maxRows: 3 }}
-              // onFocus={() => setIsDescriptionEditable(true)}
-              onBlur={() => setIsDescriptionEditable(false)}
-              bordered={false}
-              maxLength={180}
-              value={description}
-              onChange={(e) => descriptionChangeCallback(e.target.value)}
-              placeholder={descriptionPlaceholder}
-              onPressEnter={() => setIsDescriptionEditable(false)}
-            />
-          ) : (
-            <div className="editor-description">
-              <Typography.Paragraph
-                ellipsis={{
-                  rows: 3,
-                }}
-                editable={{
-                  icon:
-                    description.length > 0 ? (
-                      <RQButton
-                        onClick={() => setIsDescriptionEditable(true)}
-                        className="edit-description-btn"
-                        type="text"
-                      >
-                        Edit description
-                      </RQButton>
-                    ) : (
-                      <></>
-                    ),
-                  tooltip: false,
-                }}
-                onClick={() => setIsDescriptionEditable(true)}
-              >
-                {description ? description : descriptionPlaceholder}
-              </Typography.Paragraph>
-            </div>
+            )}
+          </Row>
+        </Col>
+        <>
+          {fileType && (
+            <Col
+              className="mock-tag editor-title-tag"
+              style={{
+                color: fileTypeColorMap[fileType],
+              }}
+            >
+              {fileType}
+            </Col>
```

Comment:
Don't pass `fileType` as this is a generic component. Make this generic so that we can use it in rule editor also if required. 
- fileType -> tagText. 
- We can keep a single color for now